src/utils/shareReply.ts

```diff
@@ -0,0 +1,19 @@
+import templite, { Values } from 'templite';
+import { log } from './log';
+import messages from './messages.json';
+
+export function share(replyName: string, values?: Values): string {
+	// @ts-ignore - Safe to ignore, there is a conditional check it
+	const reply: string = messages[replyName];
+
+	if (!reply) {
+		log(`Message ${replyName} not found in messages.json`, 'SHARE', 'red');
+		return '';
```

Comment:
```suggestion
		return replyName;
```

Returning the original key is best to spot bugs.

Reply:
![image](https://user-images.githubusercontent.com/79790819/165958210-4a53e68b-b3c5-4a08-9cce-a3f59b295c09.png)